src/vs/workbench/contrib/interactive/browser/interactive.contribution.ts

```diff
@@ -407,7 +407,38 @@ registerAction2(class extends Action2 {
 				]);
 
 				await editorControl.notebookEditor.executeNotebookCells(editorControl.notebookEditor.viewModel!.getCells({ start: index, end: index + 1 }));
-				editorControl.notebookEditor.revealCellRangeInView({ start: index, end: index + 1 });
+				const cell = editorControl.notebookEditor.getCellsFromViewRange(index, index + 1);
+				if (cell.length === 1) {
+					editorControl.notebookEditor.focusNotebookCell(cell[0], 'output');
```

Comment:
The challenging part here is `await executeNotebookCell` doesn't really guarantee that the outputs are fully rendered when the execution finishes, so a `focusNotebookCell` call after it still has a high chance to work incorrectly (reveal half of the output for example)